app/controllers/users_controller.rb

```diff
@@ -29,6 +29,21 @@ def delete_card
     redirect_to account_path
   end
 
+  def add_card 
```

Comment:
not really because that controller is a users_controller and the `create` method should be for creating a user. An alternative is to have another controller called Card controller, which would have a create method in it.